src/app/content-blocks/block-collapse/block-collapse.component.ts

```diff
@@ -6,6 +6,7 @@ import { Component, OnInit } from '@angular/core';
   styleUrls: ['./block-collapse.component.scss']
 })
 export class BlockCollapseComponent implements OnInit{
+  show: false;
```

Comment:
Would be nice to have openByDefault in the cms as a checkbox?

Reply:
Well in this point, when you click to open one they all will open, and vise versa. :)